resources/assets/coffee/react/beatmapset-page/scoreboard-tab.coffee

```diff
@@ -15,17 +15,14 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
 ###
-{span} = React.DOM
+{div} = React.DOM
 
-class BeatmapsetPage.ScoreboardTab extends React.Component
-  _scoreboardSwitch: =>
-    $.publish 'beatmapset:scoreboard:set', scoreboard: @props.scoreboard
+BeatmapsetPage.ScoreboardTab = (props) ->
+  className = 'beatmapset-scoreboard__tab page-tabs__tab'
+  className += ' page-tabs__tab--active' if props.active
 
-  render: ->
-    className = 'beatmapset-scoreboard__tab'
-    className += ' beatmapset-scoreboard__tab--active' if @props.scoreboard == @props.currentScoreboard
-
-    span
-      className: className
-      onClick: @_scoreboardSwitch
-      osu.trans "beatmaps.beatmapset.show.extra.scoreboard.#{@props.scoreboard}"
+  div
+    className: className
+    onClick: ->
+      $.publish 'beatmapset:scoreboard:set', scoreboardType: props.type
```

Comment:
this neither set `resetMods` nor `enabledMod`, resulting in `null` `enabledMods` and breaks stuff. Switch type, select a filter, and then unselect it - `enabledMods` then contains something like this:

![](https://i.imgur.com/guOe0Zk.png)